config/routes.rb

```diff
@@ -45,6 +45,7 @@
   end
 
   namespace :merchant do
+    get '/', to: 'dashboard#index'
```

Comment:
I'm not sure if this route is necessary but I added it in here in case a user wants to access /merchant or /admin